### PR TITLE
[CortexXpanse] Update Integration Fetch Offset

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-8176

## Description
We were trying to set a one hour offset before but found that the time is in epoch so this wasn't the case.  Also adding debug statements for future tshoot.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@BigEasyJ , please approve when possible

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py [hunk 1]
```diff
@@ -1416,6 +1417,14 @@ def main() -> None:
             headers=headers,
             proxy=proxy)
 
+        # To debug integratin instance configuration.
```
Comment (BigEasyJ) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py:1420]:
```suggestion
        # To debug integration instance configuration.
```

**Review by BigEasyJ — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@shmuel44  / @MLainer1 , This has been approved by Xpanse.  Please merge

**Comment by johnnywilkes** [comment]
@shmuel44 / @MLainer1 , This has been approved by Xpanse. Please merge

**Review by moishce — APPROVED** [review]

**Pull request merged**